message_definitions/v1.0/messages.xml

```diff
@@ -2463,6 +2471,13 @@
       <field name="data" type="uint8[]"/>
     </message>
 
+    <message name="DCF_REG_TABLE" id="158" link="forwarded">
+      <description>Init the table of an aircraft for the Distributed Circular Formation algorithm</description>
+      <field name="ac_id" type="uint8">ID of the table's aircraft to be updated</field>
+      <field name="nei_id" type="uint8">ID of the neighborh for the table</field>
```

Comment:
are you going to use the `nei_id = 0` to clear the table in the end ? If yes it should be in the description of the field